git_wrapper/remote.py

```diff
@@ -65,24 +67,27 @@ def fetch(self, remote="origin"):
             raise exceptions.ReferenceNotFoundException(msg)
 
         try:
-            remote.fetch()
+            remote.fetch(prune=prune, prune_tags=prune_tags)
         except git.GitCommandError as ex:
             msg = (f"Could not fetch remote {remote.name} ({remote.url}). "
                    f"Error: {ex}")
             raise exceptions.RemoteException(msg) from ex
 
-    def fetch_all(self):
+    def fetch_all(self, prune=False, prune_tags=False):
         """Refresh all the repo's remotes.
 
            All the remotes will be fetched even if one fails ; in this case a
            single exception containing the list of failed remotes is returned.
+
+           :param bool prune: True if you want to prune
+           :param bool prune_tags: True if you want to prune tags
         """
```

Comment:
Similar explanation here. Maybe it can be inspired from the man page explanation for the --prune parameter, so library consumers really understand what they're doing when they're deciding whether to use the option.

Reply:
I suppose "remote" should be plural in the explanation here, but this is good otherwise!